tool/gravity/cli/install.go

```diff
@@ -782,8 +781,6 @@ func NewServiceListener() (net.Listener, error) {
 // InterruptSignals lists signals installer service considers interrupts
 var InterruptSignals = signals.WithSignals(
 	os.Interrupt,
-	syscall.SIGTERM,
```

Comment:
I thought it was wrong afterall that stopping the installer service ( `systemctl stop <installer-service>`) would abort it (with all the consequences).